### ISSUE TITLE
Unexplainable login error

### ISSUE THREAD

**Issue by danhab99** [hfreire/facebook-login-for-robots#129]
I tried using this package for my [robot](https://github.com/danhab99/facebook99) and it wrote this:

```
Error: unable to login
    at browser.newPage.then.then (C:\Users\Dan\Documents\git\facebook99\node_modules\facebook-login-for-robots\lib\facebook-login-for-robots.js:81:19)
    at <anonymous>
```

Any explanation?

**Comment by deex2** [comment]
Same problem. Tried many things but I failed.

**Comment by rlewkowicz** [comment]
@danhab99 @deex2 

My personal issue was using a non secure redirect uri. If you add puppeteer options to your object constructor (I'm not a js guy, idk if thats the right terminology)  you can see the error messages from fb:

```
const facebookLogin = new FacebookLogin({
  facebook: {
    email: 'you@gmail.com',
    password: 'yourpass'
  },
  puppeteer: {
     headless: false // helpful for troubleshooting
  }
})
```

**Comment by danhab99** [comment]
Thank you for your recommendation, I tried re-running my test script and observed puppetters actions closely. It:

* Starts chromium
* Types in the provided email
* Types in the provided password
* Clicks login
* Redirects browser to https://www.facebook.com/dialog/oauth?client_id={MY CLIENT ID}&redirect_uri={MY REDIRECT URL]
* Facebook then complains that the redirect url does not match the authorized domains on the app registry

So I concluded that this isn't the the package's fault, more correctly it's my fault. I fixed it but I wish that hfreire/facebook-login-for-robots would report more detailed errors.
